test/Structures/Collection.spec.js

```diff
@@ -3204,5 +3204,19 @@ describe('Collection', () => {
             })
         })
     })
+
+    describe('toArray', () => {
+        it('should return an array of all toJSON models', () => {
+            let c = new Collection();
+            let m1 = c.add({id: 1});
+            let m2 = c.add({id: 2});
+
+            let cArray = c.toArray();
+
+            expect(cArray).to.be.an('array');
+            expect(cArray).to.have.lengthOf(2);
+            // expect(cArray).to.include.members([{id: 1}, {id: 2}])
```

Comment:
We should definitely have a hard expectation here. Exactly what we expect the array to contain. We should also test that an overridden `toJSON` of a model is honoured here.

Reply:
got it